cmd/thanos/rule_test.go

```diff
@@ -10,6 +10,46 @@ import (
 	"github.com/pkg/errors"
 )
 
+func Test_parseFlagLabels(t *testing.T) {
+	var tData = []struct {
+		s   []string
+		err error
```

Comment:
we use this only to specify if error is there or not. So simple `bool` would be more accurate here, right? (: 

something like `expectError bool`